welkin/models/formation.py

```diff
@@ -15,8 +15,11 @@ class FormationDataType(Enum):
 class Formations(Collection):
     iterator = FormationIterator
 
-    def get(self, data_type: FormationDataType, **kwargs):
+    def get(self, data_type: FormationDataType, params: dict = {}, *args, **kwargs):
```

Comment:
Big problem here. Don't use a mutable instance as a default argument. This can cause *huge* unexpected problems, and I say this from experience.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

If you want to codify parameters, add them as arguments here and pre-build the parameters dict from those arguments. Otherwise, the underlying `Collection.get` will handle params passed here without any extra work on your part.

Reply:
Wow yes thank you, I'll leave them out